code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -1,3 +1,6 @@
+#define ACTIVATION_COST (300 KILO JOULES)
+#define ACTIVATION_UP_KEEP (50 KILO WATTS)
```

Comment:
Make sure you scale this inversely by whatever seconds_per_tick is for whatever processes this. If it's 1 then it's fine.

Reply:
no scaling required because i use this define only when using cell charge & no where else. same for activation cost